helmfile.d/0000.prometheus-operator.yaml

```diff
@@ -28,7 +28,8 @@ releases:
   version: "0.0.29"
   wait: true
   values:
-    - rbacEnable: false
+      ### Optional: PROMETHEUS_OPERATOR_RBAC_ENABLED;
+    - rbacEnable: {{ env "PROMETHEUS_OPERATOR_RBAC_ENABLED" | default "false" }}
```

Comment:
Okay, let's do something like what you suggested. Create a generic env called `RBAC_ENABLED` and use that everywhere we have RBAC. To your point, doesn't make sense that we have to define over and over again.